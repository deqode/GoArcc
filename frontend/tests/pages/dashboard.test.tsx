import renderer from 'react-test-renderer'
import Landing from '../../pages'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<Landing url={'moc url'} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
