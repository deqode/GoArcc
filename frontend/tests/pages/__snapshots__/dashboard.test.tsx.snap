// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<div>
  <div
    className="MuiPaper-root MuiPaper-elevation0 MuiPaper-rounded"
    style={
      Object {
        "padding": "100px",
      }
    }
  />
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-4"
    >
      <h1
        className="MuiTypography-root MuiTypography-h1"
        style={
          Object {
            "fontSize": "50px",
          }
        }
      >
        Let us make your cloud work for you
      </h1>
      <h6
        className="MuiTypography-root MuiTypography-subtitle2"
        style={
          Object {
            "fontSize": "15px",
          }
        }
      >
        provide your repo, select your cloud, and let Alfred do the heavy lifting.
      </h6>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
    >
      <div
        className="text-center"
      >
        <div
          className="sign_up_head"
        >
          Sign Up
        </div>
        <a
          className="btn github_btn"
          href="moc url"
        >
          Login with github
          <img
            alt="Login with github"
            src="/assets/github_icon.png"
          />
        </a>
      </div>
    </div>
  </div>
  <div
    className="MuiPaper-root MuiPaper-elevation1 MuiPaper-rounded"
  />
</div>
`;
